configure.ac

```diff
@@ -317,6 +317,20 @@ AM_CONDITIONAL([MAINTAINER_TOOLS],[test x"${i_do_have_maintainer_tools}" = xyes]
 AC_SUBST([MAINTAINER_TOOLS],["${i_do_have_maintainer_tools}"])
 
 #--------------------------------------------------------------------------
+# Check for autotrace and potrace (don't do anything with this yet)
```

Comment:
@JoesCat, as indicated before, my primary concern is that this is actually more misleading than it's meant to help. See my comment wrt. it being confused as actually affecting the build during compile time. Certainly when I first saw the check, my first thought was *what part of the build now relies on autotrace/potrace*?

I'm in favour of removing this check altogether as it just doesn't belong in the build system.